OpenRobertaServer/staticResources/js/main.js

```diff
@@ -62,6 +62,7 @@ require.config({
         'socket.controller': 'app/roberta/controller/socket.controller',
         'webview.controller': 'app/roberta/controller/webview.controller',
         'sourceCodeEditor.controller': 'app/roberta/controller/sourceCodeEditor.controller',
+        'orb.model' : '../app/roberta/models/orb.model',
```

Comment:
Since this is not used, and doesn't seem to function correctly remove this line too.